README.md

```diff
@@ -1 +1,73 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
Please change this to the actual technologies used. In your case `HTML` and `CSS`